go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250126142519-d50affd316d3
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250127080549-a0bcacd77bd8
```

Comment:
Commenting to remember to update the operator version after the operator PR is merged and before merging this PR